apps/prairielearn/src/pages/administratorCourseRequests/administratorCourseRequests.ts

```diff
@@ -10,24 +9,29 @@ import {
   updateCourseRequest,
 } from '../../lib/course-request';
 import { selectAllInstitutions } from '../../models/institution';
+import { AdministratorCourseRequests } from './administratorCourseRequests.html';
 
 const router = express.Router();
 
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    res.locals.coursesRoot = config.coursesRoot;
-    res.locals.course_requests = await selectAllCourseRequests();
-    res.locals.institutions = await selectAllInstitutions();
-    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    const rows = await selectAllCourseRequests();
+    const institutions = await selectAllInstitutions();
+    res.send(
+      AdministratorCourseRequests({
+        rows,
+        institutions,
+        coursesRoot: config.coursesRoot,
+        resLocals: res.locals,
+      }),
+    );
   }),
 );
 
 router.post(
   '/',
   asyncHandler(async (req, res) => {
-    if (!res.locals.is_administrator) throw error.make(403, 'Insufficient permissions');
-
```

Comment:
AFAICT this permissions check is unnecessary; we already have middleware sitting in front of all admin routes that checks for this.